packages/terra-data-grid/src/subcomponents/ColumnResizeHandle.module.scss

```diff
@@ -4,22 +4,23 @@
 
 :local {
   .resize-handle {
-    border-right: 2px solid transparent;
+    border-right: 4px solid transparent;
     cursor: col-resize;
     display: block;
+    height: 100%;
     position: absolute;
     right: 0;
     top: 0;
-    width: 24px;
+    width: 22px;
     z-index: 1;
-
+    
     &:focus {
-      border-color: var(--terra-data-grid-resize-handle-focus-border-color, rgba(34, 42, 46, 1)) !important;
+      border-color: var(--terra-data-grid-resize-handle-focus-border-color, #007cc3) !important;
       outline: 0 !important;
     }
 
     &:hover {
-      border-color: var(--terra-data-grid-resize-handle-hover-border-color, rgba(34, 42, 46, 0.6));
+      border-color: var(--terra-data-grid-resize-handle-hover-border-color, #909496);
```

Comment:
There should not be a need to change from RGBA to HEX.  Also, I am not sure that this conversion is correctly accounting for opacity.

Reply:
I thought it was odd that this was using RGBA vs HEX in the other stylesheets but you're right about the opacity. Reverting it for now.

Reply:
Removed here: 010c3f9825fddd05ed9c04a188bf0c8a422f0df3